northstar_tests/test_container/manifest.yaml

```diff
@@ -3,6 +3,10 @@ version: 0.0.1
 init: /test_container
 uid: 1000
 gid: 1000
+# cgroups:
+#   memory:
+#     limit_in_bytes: 10000000
+#     swappiness: 0
```

Comment:
what happened to these configs?

Reply:
Commented along the the cgroup mem test. See above.

Reply:
Created #309